serve/mlc_serve/model/paged_cache_model.py

```diff
@@ -335,13 +336,207 @@ def get_tvm_model(artifact_path, model, quantization, num_shards, dev):
 
     lib_path = os.path.join(model_artifact_path, f"{model}-{quantization}-cuda.so")
 
+    chat_file = os.path.join(f"{model_artifact_path}/params", "mlc-chat-config.json")
```

Comment:
> But I'm wondering if we can introduce some mlc-serve configs to get rid of the dependency on hf config.

This is exactly done in this PR by switching to mlc_chat_config.json in `paged_cache_model.py`. The `mlc_chat_config.json` originally is created for single batch `cpp/llm_chat.cc` but it would be nice to reuse it instead to introduce one more config file during the build of the model. But if you consider that it make sense to create separate config for mlc-serve, we can duplicate it during model compilation